CHANGELOG.adoc

```diff
@@ -5,6 +5,28 @@ https://spring.io/projects/spring-cloud-gcp[Spring Cloud GCP] is a set of integr
 This document provides a high-level overview of the changes introduced in Spring Cloud GCP by release.
 For a detailed view of what has changed, refer to the https://github.com/GoogleCloudPlatform/spring-cloud-gcp/commits/main[commit history] on GitHub.
 
+== 2.0.3 (2021-06-08)
```

Comment:
```suggestion
== 2.0.3 (2021-06-09)
```

Reply:
Interesting question; technically the release jars went out yesterday.

Reply:
I didn't realize, I take that back.